.travis.yml

```diff
@@ -20,6 +20,8 @@ install:
 
 jobs:
   include:
+    - stage: Prettier Diff
+      script: npm run prettier:diff
```

Comment:
To confirm, `--list-different` returns an exit code if there are any different files?